### ISSUE TITLE
[Feature Request][Mobile] Support "Run from here" for Mobile?

### ISSUE THREAD

**Issue by thongnmtran** [katalon-studio/katalon-studio#304]
From Katalon Forum: https://forum.katalon.com/t/run-from-here-in-mobile/41308/4
> "Testing is ongoing but this feature would be very useful since loading apps typically takes a couple of minutes each time and then another couple of minutes to log in and sync data"

---

Request Feature: Support to **`Run from here`** for Mobile just like with the Web Recorder
![image](https://user-images.githubusercontent.com/46991507/77608738-a8e70280-6f50-11ea-97f4-96defd54afec.png)

**Comment by martin-mtx** [comment]
Is there an approximate time when this might become available?

**Comment by natesthompson** [comment]
Yes I would like to know an approximate time as well!  This would be a great feature to have.
